app/services/register_records_downloader.rb

```diff
@@ -20,7 +20,7 @@ def download_format(format)
   end
 
   def download_ods
-    ods_download_to_ga
+    #ods_download_to_ga
```

Comment:
for all of the GA stuff, does it make sense to delete the code instead? This way we can restore them with a quick git-revert, and no dead code lying around.